packages/contracts-bedrock/contracts/L2/BVM_ETH.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+/* Library Imports */
+import { Predeploys } from "../libraries/Predeploys.sol";
+
+/* Contract Imports */
+import { MantleMintableERC20 } from "../universal/MantleMintableERC20.sol";
+
+/**
+ * @title BVM_ETH
+ * @dev The ETH predeploy provides an ERC20 interface for ETH deposited to Layer 2. Note that
+ * unlike on Layer 1, Layer 2 accounts do not have a balance field.
+ */
+contract BVM_ETH is MantleMintableERC20 {
```

Comment:
Is this contract just for test? If so please move it under test directory.

Reply:
it is used for dev-network up